opendss/opendss.md

```diff
@@ -5,14 +5,27 @@ nav_order: 6
 ---
 # OpenDSS Converter
 
-
 Visit the [OpenDSS Installation page](../installation/ditto_reader.md) to install OpenDSS and URBANopt DiTTo Reader.
 
 ## Usage
 
-Follow these steps to run the OpenDSS workflow via the CLI:
+The DiTTo-Reader/OpenDSS workflow is available via the `opendss` URBANopt CLI command.
+
+### Notes
+
+- The feature file should contain Electrical Connectors and Junctions for a successful OpenDSS run.
+- If you want to include reopt results as an input to OpenDSS, make sure to include the --reopt flag when issuing the `opendss` command.
+- Once the `opendss` command has been run, the `process --opendss` command can then be run to pull the opendss results back into the URBANopt reports.
+
+## Usage
+
+For in-terminal help:
+```bash
+uo opendss --help
+```
 
-1. Create and run your project as you normal would, following the [usage examples](../usage/run_project.html#urbanopt-cli-usage-examples). Ensure you use a [feature file](../overview/definitions.md#feature-file) that includes the necessary electrical network configuration (e.g. `example_project_with_electric_network.json`).
+An example:
+1. Create and run your project as you normal would, following the [usage examples](../usage/run_project.html#urbanopt-cli-usage-examples).
```

Comment:
add "make sure you create a project with electrical lines" and give the command for that